experiments/compositions/composition/config/expanders/expander_versions.yaml

```diff
@@ -16,12 +16,27 @@ apiVersion: composition.google.com/v1alpha1
 kind: ExpanderVersion
 metadata:
   name: jinja2
-  namespace: composition-system
+  namespace: system
 spec:
+  type: job
   imageRegistry: gcr.io/krmapihosting-release
   validVersions:
   # current golang semver package only support version comparison
   # in the format of `v{num}.{num}.{num}-{alphabet}`, `v{num}.{num}.{num}` and `{num}.{num}.{num}`.
   # Removing the `.alpha` here for now. Will be added back in the test.
   - v0.0.1
+  - v0.0.0
+---
+apiVersion: composition.google.com/v1alpha1
+kind: ExpanderVersion
+metadata:
+  name: gjinja2
```

Comment:
Is the name supposed to be jinja2?

Reply:
No.
To differentiate job/grpc based expanders iam calling it gjinja2.
At some point soon i will set this to jinja and the pod based one to something else.

Reply:
Thanks for the explanation!